lib/taskana-core/src/main/java/pro/taskana/classification/api/ClassificationService.java

```diff
@@ -5,12 +5,16 @@
 import pro.taskana.classification.api.exceptions.ClassificationNotFoundException;
 import pro.taskana.classification.api.exceptions.MalformedServiceLevelException;
 import pro.taskana.classification.api.models.Classification;
+import pro.taskana.common.api.TaskanaRole;
 import pro.taskana.common.api.exceptions.ConcurrencyException;
 import pro.taskana.common.api.exceptions.DomainNotFoundException;
 import pro.taskana.common.api.exceptions.InvalidArgumentException;
 import pro.taskana.common.api.exceptions.NotAuthorizedException;
+import pro.taskana.task.api.models.Task;
 
-/** This class manages {@linkplain Classification Classifications}. */
+/**
+ * The ClassificationService manages all operations on {@linkplain Classification Classifications}.
```

Comment:
We don't have a clear rule for linking classes when referencing multiple of them. 
Let's talk about this scenario, reach an agreement and update our guidelines. 

No need to act here for now.

Reply:
Ok :)